android/expoview/src/main/java/host/exp/exponent/experience/ReactNativeActivity.kt

```diff
@@ -610,8 +611,8 @@ abstract class ReactNativeActivity :
     )
   }
 
-  val devSupportManager: RNObject
-    get() = reactInstanceManager.callRecursive("getDevSupportManager")!!
+  val devSupportManager: RNObject?
+    get() = if (reactInstanceManager.isNotNull) reactInstanceManager.callRecursive("getDevSupportManager") else null
```

Comment:
```suggestion
    get() = reactInstanceManager?.callRecursive("getDevSupportManager")
```

Reply:
was curious about how `RNObject.isNotNull` interop with kotlin nullable?
https://github.com/expo/expo/blob/83f8eb56baf0a28a66611f6795187e05cd09d27f/android/expoview/src/main/java/host/exp/exponent/RNObject.kt#L33-L34

`RNObject.isNotNull` indicates its internal instance to be null. however, `RNObject?` is the RNObject to be null.

Reply:
```suggestion
    get() = reactInstanceManager.takeIf { it.isNotNull }?.callRecursive("getDevSupportManager")
```
This should work too 😁